docs/kyma-environment-broker/03-12-cls-account-setup.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Cloud Logging Service Account setup
+type: Details
+---
+
+The CLS instances are created inside a subaccount inside the Kyma global account: When the customer provisions the first SKR instance (SKR 1) for their SKR global account, one CLS instance is created inside the Kyma global account.
+
+The following image explains the account setup for SKR and CLS instances:
+
+![CLS Account diagram](./assets/cls-acc.jpeg)
+
+The diagram illustrates the following: 
+1. When a customer with global account X provisions the first SKR (SKR 1), a CLS instance (CLS for customer X) is created in the CLS subaccount that is part of the Kyma global account. 
```

Comment:
The list doesn't seem to be formatted properly in the preview: 

<img width="1547" alt="Screen Shot 2021-03-15 at 16 27 18" src="https://user-images.githubusercontent.com/49867570/111178415-68422800-85ab-11eb-9447-be1d305960e5.png">